test/unit/vm/compile_sass_test.dart

```diff
@@ -25,6 +25,10 @@ void main() {
     const defaultSourceDir = 'test/unit/vm/fixtures/sass/';
     const specificOutputDir = 'test/unit/vm/fixtures/css/';
 
+    setUp(() {
```

Comment:
Is it possible to add a test for the --watch option? You'd probably have to have a way to kill the execution of `main` from inside the test asynchronously, and I'm not sure if that's possible to do programatically.

Reply:
I couldn't figure out a way.